E3Next/Utility/e3util.cs

```diff
@@ -65,6 +65,12 @@ public static void PutOriginalTargetBackIfNeeded(Int32 targetid)
                 }
             }
         }
+
+        public static Boolean CompareInsensitive(this string str, string compareStr)
```

Comment:
change to EqualsIns , as its an equal operator and insensitive is a large word. Otherwise exactly what I meant

Reply:
Wee, now I know how to make categories in C#

Reply:
or string.eqic(user)

Reply:
Happy to call it whatever, even string.RekkaEqic(user) :)